### DISCUSSION TITLE
How can i limit the particles

### DISCUSSION THREAD

**Discussion by FoggY9** [smaranjitghose/awesome-portfolio-websites#1295] [Q&A] [answered]
they are huge, and if someone clicks it too much it makes more. and at a time the page becomes too heavy. how can i limit particle generating

**Comment by anushbhatia** [comment, marked as answer]
https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/assets/js/particle.js
Yes here's the link to the particle js section
